docassemble/ALToolbox/test_al_income.py

```diff
@@ -0,0 +1,153 @@
+import unittest
+
+from decimal import Decimal
+from .al_income import (
+    ALIncome,
+    ALIncomeList,
+    ALAsset,
+    ALAssetList,
+    ALVehicle,
+    ALVehicleList,
+    ALSimpleValue,
+    ALSimpleValueList,
+    ALItemizedJob,
+    ALItemizedValueDict,
+    ALItemizedValue,
+    recent_years,
```

Comment:
Is this used in here now? Or is it planned for future use?

Reply:
Future use. Would like most of the things to be tested, but not sure how to make a tests that doesn't just duplicate the function itself.